Makefile

```diff
@@ -38,8 +38,6 @@ lint:
 	mypy --strict samtranslator bin
 	# Linter performs static analysis to catch latent bugs
 	pylint --rcfile .pylintrc samtranslator
-	# Ensure templates adhere to JSON schema
-	bin/validate_schema.py
```

Comment:
The file should be deleted as well.

Reply:
Oh I thought I deleted it. My force push must have reverted this change...